### DISCUSSION TITLE
How to read just the dictionary page for a column having single dictionary page and rest RLE encoding data pages

### DISCUSSION THREAD

**Discussion by erchirag** [G-Research/ParquetSharp#338] [Q&A] [answered]
To do quick filtering based on dictionary page,  I want to just read dictionary page and check it against set of values, how can I do this ?

**Comment by adamreeve** [comment, marked as answer]
Hi Chirag

This isn't currently possible with ParquetSharp, and I don't believe that the Arrow C++ Parquet library exposes the dictionary data either. You should be able to access the min and max values of a column within each row group via the column statistics though, would that work for you?

**Reply by erchirag** [reply]
Thank Adam, I was hoping to use it to speed up column search for columns which only are completely dictionary encoded. Its fine good to know it is not possible so I stop searching through code and .net reflection :)
